src/test/java/spoon/test/compilationunit/GetBinaryFilesTest.java

```diff
@@ -115,7 +115,7 @@ public void testNestedTypes() throws IOException {
 
 	@Test
 	public void testAnonymousClasses() throws IOException {
-		final String input = "./src/test/java/spoon/test/secondaryclasses/AnonymousClass.java";
+		final String input = "./src/test/java/spoon/test/secondaryclasses/testclasses/AnonymousClass.java";
```

Comment:
The origin line was correct. Please check it again

Reply:
ok, I see. You moved classes into different package. Why?

Reply:
To exclude them from analysis succinctly here: https://github.com/INRIA/spoon/pull/1924/files#diff-600376dffeb79835ede4a0b285078036R296